Controller/Adminhtml/Landingpage/Index.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\Controller\Adminhtml\Landingpage;
+
+use Magento\Framework\Controller\ResultFactory;
+
+class Index extends AbstractAction
+{
+    /** @return \Magento\Framework\View\Result\Page */
+    public function execute()
+    {
+        $breadMain = __('Algolia | Landing Page Builder');
+
+        /** @var \Magento\Framework\View\Result\Page $resultPage */
+        $resultPage = $this->resultFactory->create(ResultFactory::TYPE_PAGE);
+        $resultPage->setActiveMenu('Algolia_AlgoliaSearch::manage');
+        $resultPage->getConfig()->getTitle()->prepend($breadMain);
+
+        $dataPersistor = $this->_objectManager->get(\Magento\Framework\App\Request\DataPersistorInterface::class);
+        $dataPersistor->clear('landing_page');
```

Comment:
What is data persistor?

Reply:
I guess it's used to keep the submitted data when needed : https://github.com/algolia/algoliasearch-magento-2/blob/feat-landing-page-builder/Controller/Adminhtml/Landingpage/Save.php#L119.
I took inspiration from the M2 CMS page feature for this : https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Cms/Controller/Adminhtml/Page/Index.php#L56